src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -483,6 +483,7 @@ public List<OffenderPropertyContainer> getActivePropertyContainers() {
 
     public List<OffenderProfileDetail> getActiveProfileDetails() {
         return profileDetails.stream()
+            .filter(pd -> Objects.nonNull(pd.getProfileCode()))
```

Comment:
not really needed since when this is used the nulls are filtered out anyway by the caller, but given the method name  this at least means this method behaves the same after the mapping change below